src/dfcx_scrapi/tools/google_drive_connector.py

```diff
@@ -0,0 +1,196 @@
+"""Utility file for Google Sheets"""
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import time
+
+import google.auth
+import gspread
+from gspread_dataframe import set_with_dataframe
+from oauth2client.service_account import ServiceAccountCredentials
+import pandas as pd
+
+
+class GoogleDriveConnector:
+    """Class for google sheets operations"""
+
+    global_scopes = [
+        "https://spreadsheets.google.com/feeds",
+        "https://www.googleapis.com/auth/drive",
+    ]
+
+    def __init__(
+        self, creds_path: str = None, creds_dict: dict = None, scope=False
+    ):
+        scopes = GoogleDriveConnector.global_scopes
+        if scope:
+            scopes += scope
+
+        if creds_path:
+            creds = ServiceAccountCredentials.from_json_keyfile_name(
+                filename=creds_path, scopes=scopes
+            )
+        elif creds_dict:
+            creds = ServiceAccountCredentials.from_json_keyfile_dict(
+                keyfile_dict=creds_dict, scopes=scopes
+            )
+        else:
+            creds = google.auth.default(scopes=scopes)[0]
+
+        self.client = gspread.authorize(creds)
+
+    def create_google_sheet(self, sheet_name: str):
```

Comment:
@Hgithubacct Can you add a return or log statement here that fetches the created Google Sheet and provides the hyperlink to the user?

This would be a good quality of life improvement so the user doesn't have to:
- Call `create_google_sheet(sheet_name = 'My Cool Sheet')`
- Browse to Google Sheets / Google Drive to find their newly created Sheet

It would provide a 1-click link from wherever they are working at to use at their discretion.